aws-java-sdk-core/src/main/java/com/amazonaws/retry/PredefinedBackoffStrategies.java

```diff
@@ -147,4 +144,10 @@ public final long delayBeforeNextRetry(AmazonWebServiceRequest originalRequest,
         }
     }
 
-}
\ No newline at end of file
+    private static final long exponentialDelayHelper(int retriesAttempted, int baseDelay, int maxBackoffTime) {
```

Comment:
/nitpick - can we rename this method so that it's a verb? Something like "calculateExponentialDelay"